tests/integration/conftest.py

```diff
@@ -153,9 +153,8 @@ def run_trino():
         stop_trino(container_id, proc)
 
 
-@pytest.fixture(scope="module")
```

Comment:
Hah, I had recently learnt about pytest fixtures and ended up overusing them. 😛 

Thanks for the cleanup.

Reply:
Yeah, fixtures cannot be used in `@pytest.mark.skipif`, so that's why I converted it. 

Squash done.